publish-js.sh

```diff
@@ -35,6 +35,14 @@ echo "Running buildjs script"
 ./amf-client/js/build-scripts/buildjs.sh
 echo "Finished buildjs script"
 
+echo "Generating typings"
+sbt -Dsbt.sourcemode=true clientJS/generateTypings
```

Comment:
To be able to read source files, the typings have to created with source mode on.

Reply:
This might be error prone in our release process, we should leave a reminder in our release documentation that aml and core repositories be aligned with the amf sources that are being published.

Reply:
Yup, agree. Will update it